src/libs/auth.js

```diff
@@ -168,7 +168,7 @@ authStore.subscribe((state, oldState) => {
 
 export const refreshTerraProfile = async () => {
   const profile = Utils.kvArrayToObject((await Ajax().User.profile.get()).keyValuePairs)
-  authStore.update(state => _.set('profile', profile, state))
+  authStore.update(state => ({ ...state, profile }))
```

Comment:
TOL: I'm curious why you went this way and not `_.set('profile', profile)`?

Reply:
Consistency with other uses in the file.